File: Project/Blazor_pwa/Models/Implementations/Animation.cs
namespace Blazor_pwa.Models.Implementations
{
    public class Animation
    {
        public string Name { get; set; }
        public Dictionary<string, string> Transform { get; private set; } = new();

        public Animation(string name)
        {
            Name = name;
            Transform.Add("translateX", "0px");
            Transform.Add("translateY", "0px");
            Transform.Add("scaleX", "1");
            Transform.Add("scaleY", "1");
            Transform.Add("rotate", "0deg");
        }
    }
}

File: Project/Remix_pwa/app/Models/implementations/Animation.ts
export class MyAnimation {
    _name: string = "animation";
    _transform: Map<string, string> = new Map<string, string>();

    get Transform(): Map<string, string> {
        return this._transform;
    }

    set Name(name: string) {
        this._name = name;
    }

    get Name(): string {
        return this._name;
    }

    constructor() {
        this._transform.set("translateX", "0px");
        this._transform.set("translateY", "0px");
        this._transform.set("scaleX", "1");
        this._transform.set("scaleY", "1");
        this._transform.set("rotate", "0deg");
    }

    get ToString(): string {
        return `Animation '${this._name}'`;
    }
}